portfolio/src/main/webapp/oauth.js

```diff
@@ -0,0 +1,16 @@
+
+function signIn(googleUser) {
+  const profile = googleUser.getBasicProfile();
+  const imagurl=profile.getImageUrl();
```

Comment:
please apply the javascript style guide (see the section on spacing in assignments) also the the rest of the function.

Reply:
I'm sorry! Previously I didn't see this.